templates/Segment_Image_Data.ipynb

```diff
@@ -344,13 +344,13 @@
    ],
```

Comment:
<p>Instead of indexing into data_xr_summed to get the FOVs list, I think we should create a variable called display_fov and set it equal to "Point8", and then pass that into visualize_segmentation</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/357/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>